### PR TITLE
Chatbot covid pmidc team rmoupdate

### PR DESCRIPTION
**Pull request by egovmis1**
In rmo , a third option is added. @rushang7-eGov @ramakrishnaippili-eGov @rakesh-wt-egov

### CODE REVIEW

File: xstate-chatbot/nodejs/src/machine/service/cova-vitals-service.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ class VitalsService {
     return { response: 0 };
   }
 
-  async getMoSubmitReport(caseid, mophone, remarks, specRequire, hospRequire) {
+  async getMoSubmitReport(caseid, mophone, remarks, specRequire, hospRequire, isPatientStable) {
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/service/cova-vitals-service.js:194]: Make same change in dummy service

File: xstate-chatbot/nodejs/src/machine/vitals.js [hunk 2]
```diff
@@ -133,6 +134,18 @@ const vitalsFlow = {
               actions: assign((context, event) => {
                 context.slots.person.isHospitalizationRequired = 'yes';
                 context.slots.person.isSpecializationRequired = 'no';
+                context.slots.person.isPatientStable='no';
+              }),
+              target: '#doctorRemark',
+            },
+            ,
+            {
+              cond: (context) => context.intention == 'patientStable',
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/vitals.js:143]: If the target is not different, no need to write separate cond:.
You can assign different values to different slots within the same actions function. 
Write the first condition for INTENTION_UNKNOWN which will take the user to the error state.

File: xstate-chatbot/nodejs/src/machine/vitals.js [hunk 3]
```diff
@@ -155,6 +168,7 @@ const vitalsFlow = {
       states: {
         prompt: {
           onEntry: assign((context, event) => {
+            console.log(context.slots.person)
```
Comment (rushang7-eGov) [inline xstate-chatbot/nodejs/src/machine/vitals.js:171]: Remove console log statements

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: xstate-chatbot/nodejs/src/machine/vitals.js (hunk 3 above)
Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js]: removed

File: xstate-chatbot/nodejs/src/machine/vitals.js (hunk 2 above)
Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/vitals.js:143]: actually for all three options, different different three apis are called.

File: xstate-chatbot/nodejs/src/machine/service/cova-vitals-service.js (hunk 1 above)
Comment (egovmis1) [reply xstate-chatbot/nodejs/src/machine/service/cova-vitals-service.js:194]: Chatbot covid pmidc team rmoflow #787

File: xstate-chatbot/nodejs/src/machine/vitals.js (hunk 2 above)
Comment (rushang7-eGov) [reply xstate-chatbot/nodejs/src/machine/vitals.js:143]: Still, I think there isn't any need to make separate conditions.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

**Comment by egovmis1** [comment]
Not required to merge

**Pull request closed without merging**